website/blog/2024-04-22-0.74-yoga-improvements-preparing-for-the-new-architecture.md

```diff
@@ -0,0 +1,441 @@
+        height: 25,
+        zIndex: 11,
+        position: 'absolute',
+      }}
+    />
+  </View>
+</View>
+```
+
+</td>
+<td width="300">
+
+![Static zIndex Example](../static/blog/assets/0.74-zindex-example.png)
+
+</td>
+</tr>
+</table>
+
+React Native continues to default to `position: 'relative'` when no `position` is set.
+
+### New Architecture: Bridgeless by Default
+
+In this release, we are making **Bridgeless Mode the default when the New Architecture is enabled**. You can learn more about our switch to Bridgeless as the default in this post (_coming soon_). To make the transition smoother we enhanced the interop layers to cover Bridgeless and worked with several libraries to make sure they will work in Bridgeless from day one.
+
+Bridgeless is not the only interop layer we worked on: we improved the New Renderer Interop layer too!
+
+<!-- TODO: Reinsert above - "Check out the new features [here](link_to_the_post)." -->
+
+Finally, if you want to learn more about the New Architecture in general, we moved the complete documentation from the React Native website to the [react-native-new-architecture repo](https://github.com/reactwg/react-native-new-architecture/tree/main/docs).
+
+### New Architecture: Batched `onLayout` updates
+
+State updates in `onLayout` callbacks are now batched. Previously, each state update in the `onLayout` event would result in a new render commit.
+
+```jsx
+function MyComponent(props) {
+  const [state1, setState1] = useState(false);
+  const [state2, setState2] = useState(false);
+
+  return (
+    <View>
+      <View
+        onLayout={() => {
+          setState1(true);
+       }}>
+      <View
+         onLayout={() => {
+          // When this event is executed, state1's new value is no longer observable here.
+          setState2(true);
+        }}>
+      </View>
+    </View>
+  );
+}
+```
+
+In 0.74, `setState1` and `setState2` updates are batched together. This change is [expected behavior in React](https://react.dev/learn/queueing-a-series-of-state-updates#react-batches-state-updates) and allows for less re-renders.
+
+:::danger
+This change **may break code** that has relied on un-batched state updates. You'll need to refactor this code to use [updater functions](https://react.dev/learn/queueing-a-series-of-state-updates#updating-the-same-state-multiple-times-before-the-next-render) or equivalent.
+:::
+
+### Android SDK minimum bump (Android 6.0)
+
+React Native 0.74 has a minimum SDK version requirement of 23 (Android 6.0). See our justification for this change [here](https://github.com/react-native-community/discussions-and-proposals/discussions/740).
+
+#### Bonus: Android app size reduction
+
+The minimum SDK bump, together with several improvements at our native build, allowed us to greatly reduce the app size on user devices.
+
+For example a newly created app with React Native 0.74 occupies ~13% less space on user device, resulting in ~4Mb saved on device (see the attached screenshot).
```

Comment:
```suggestion
By bumping the minimum Android SDK version to 23, we were able to land optimizations to greatly reduce the app size on user devices. For a newly created app, React Native 0.74 occupies ~13% less space on user device, resulting in ~4Mb saved on device:
```

Also, where is the attachment?

Reply:
On @huntie desktop 😅  Will arrive on Monday